metube/umbrel-app.yml

```
@@ -0,0 +1,22 @@
+manifestVersion: 1
+id: metube
+category: media
+name: MeTube
+version: "2023.11.16"
+tagline: Simple, youtube (and more) downloader web UI
+description: >-
+  Web GUI for youtube-dl with playlist support. 
+  Allows you to download videos from YouTube and dozens of other sites.
```

Shall we tweak slightly to something like the below? I'm thinking we can remove reference to it being a "web UI" like the project's github does, since user's will just be thinking about it as an app.

```suggestion
tagline: Download videos from YouTube
description: >-
  Download videos from YouTube (and dozens of other sites) directly to you Umbrel.
```